integration/testdata/main_test.go

```diff
@@ -0,0 +1 @@
+package testdata
```

Comment:
I don't think testdata should be a package. If this is because hack/make/test-integration tries to run this as a test package how about we just add testdata to the exclude list?